heroku-16-build/bin/heroku-16-build.sh

```diff
@@ -9,6 +9,7 @@ apt-get install -y --force-yes \
     autoconf \
     bison \
     build-essential \
+    gettext \
```

Comment:
Hi, and thanks for the contribution!

One question. Putting the actual `gettext` package here will install the `.so` files in the `heroku-16-build` image, but not in the `heroku-16` image. The library will be available at build-time only. Is this intended?

Reply:
Yes, one should compile gettext files only during deploy not during runtime.